samples/asynctests/test_azure_event_hubs_receive_async.py

```diff
@@ -131,7 +131,6 @@ def on_message_received(message):
 
 @pytest.mark.asyncio
 async def test_event_hubs_shared_connection_async(live_eventhub_config):
-    pytest.skip("Unstable on OSX and Linux - need to fix")  # TODO
```

Comment:
Awesome that these are now resolved :)
Once this is merged I think we have an issue for these that we can now close?